src/emulator/auth/handlers.ts

```diff
@@ -201,7 +201,7 @@ export function registerHandlers(
           <span class="mdc-list-item__text"><span class="mdc-list-item__primary-text">${
             info.displayName || "(No display name)"
           }</span>
-          <span class="mdc-list-item__secondary-text fallback-secondary-text">${
+          <span class="mdc-list-item__secondary-text fallback-secondary-text" id="reuse-email">${
```

Comment:
This ID is not unique because there will be one element for each existing `providerInfo`. Please find some other way of extracting the email, for example by modifying the logic that parses `data-id-token`

Reply:
I think a more reliable way is to modify `finishWithUser` to take an additional email argument and modify https://github.com/firebase/firebase-tools/blob/da585894ef1e226ffed49af7b0dca6987cba9b25/src/emulator/auth/widget_ui.ts#L160 and https://github.com/firebase/firebase-tools/blob/da585894ef1e226ffed49af7b0dca6987cba9b25/src/emulator/auth/widget_ui.ts#L237 to pass in the email.